running-coreos/cloud-providers/interoute/index.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: docs
+title: Interoute VDC
+category: running_coreos
+sub_category: cloud_provider
+weight: 11
+---
+
+# Running CoreOS on Interoute VDC
+
+Interoute Communications Limited is the owner operator of Europe's largest cloud services platform, which encompasses over 67,000 km of lit fiber, 12 data centres, 13 Virtual Data Centres and 31 colocation centres, with connections to 195 additional third-party data centres across Europe. 
```

Comment:
Can you remove the marketing aspects of this? And mention that it's a Cloudstack powered cloud? Something like:

```
Interoute VDC is Europe's largest cloud services platform, which is powered by Apache Cloudstack.
```